addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3482,7 +3482,10 @@ msgctxt "#800"
 msgid "Music library needs to rescan tags from files. Would you like to scan now?"
 msgstr ""
 
-#empty string with id 801
+#: xbmc/music/windows/GUIWindowMusicBase.cpp
+msgctxt "#801"
+msgid "Fetch additional information for albums and artists?[CR]This could take some time so you may prefer to do this later"
```

Comment:
Also do not use strings below 10000. If possible also move the other one to a higher number cause we need to get rid of the lower ID